drivers/volume/mock/mock.go

```diff
@@ -251,6 +251,11 @@ func (m *Driver) OwnsPVC(pvc *v1.PersistentVolumeClaim) bool {
 	return true
 }
 
+// OwnsPV returns true because it owns all PVC created by tests
+func (m *Driver) OwnsPV(pv *v1.PersistentVolume) bool {
```

Comment:
Not sure why we have this function if it always returns true? will this change in the future?

Reply:
This is a mock driver which is only used for testing. This is a part of the driver interface, each driver returns the correct value. For unit tests we use the mock driver which always return true